src/System.Runtime.Serialization.Formatters/tests/System.Runtime.Serialization.Formatters.Tests.csproj

```diff
@@ -37,6 +38,9 @@
   <ItemGroup>
     <EmbeddedResource Include="BinaryFormatterTests.rd.xml" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
+    <Reference Include="System.Text.Json" />
```

Comment:
Why isn't this a ProjectReference?

Reply:
You shouldn't need this as we auto-reference all assemblies in the runtime directory.